yaspin/core.py

```diff
@@ -319,9 +319,12 @@ def write(self, text):
             sys.stdout.write("\r")
             self._clear_line()
 
-            _text = to_unicode(text)
-            if PY2:
-                _text = _text.encode(ENCODING)
+            if isinstance(text, (builtin_str, str, bytes)):
```

Comment:
As both Py2 and Py3 has `builtin_str = str`, I believe mentioning `str` in types tuple is redundant:
```suggestion
            if isinstance(text, (builtin_str, bytes)):
```

Reply:
For PY3 this is correct, for PY2 it is not:
```
if PY2:
    builtin_str = str
    bytes = str
    str = unicode  # noqa
```
So for PY2 this is essentially `isinstance(text, (str, unicode, bytes))`.

I actually amended this change, see [the diff of my force push](https://github.com/pavdmyt/yaspin/compare/84ae048dda411f4c894268cd323860c37713a29b..347fad066cc5405bd97991e9386a4b864daa5b37). My original version worked on PY3 and broke on PY2.

Reply:
Yeah, it seems I got confused by all of that overrides in `compat.py`.